cloudping.py

```diff
@@ -38,17 +40,13 @@ def ping(event, context):
 
     print(json.dumps({
         'cloudping_result': result_value,
+        'response_time': response_time,
         'url': url,
         'options': options
     }))
 
     client = boto3.client('cloudwatch')
-    response = client.put_metric_data(
-        Namespace='cloudping',
-        MetricData=[
-            {
-                'MetricName': 'status',
-                'Dimensions': [
+    dimensions = [
                     {
```

Comment:
Please fix the indentation level for dimensions

Reply:
I have no idea what the correct indentation level would be.